requirements.txt

```diff
@@ -6,3 +6,5 @@ matplotlib
 -e .
 tqdm
 onager
+sympy
```

Comment:
Added sympy for independence testing because I couldn't figure out an elegant way to do this in numpy.